src/style.css

```diff
@@ -0,0 +1,151 @@
+/* #region normalise */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+/* endregion */
+
+/* #region productcard */
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  padding: 32px 16px 0;
+
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+
+  background: #fff;
+  border: 1px solid #f3f3f5;
+  border-radius: 5px;
+
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --button-color: #00acdc;
+}
+
+.product-card__image {
+  margin: 0 auto;
+
+  height: 134px;
+  width: 160px;
+}
+
+.product-card__product-name {
+  margin-top: 40px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.product-card__product-code {
+  margin-top: 4px;
+
+  font-weight: 400px;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-color);
+}
+
+.product-card__product-rating {
+  margin-top: 16px;
+
+  display: flex;
+  justify-content: space-between;
+}
+
+.product-card__reviews-number {
+
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.product-card__pricing {
+  margin-top: 24px;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+
+  line-height: 18px;
+}
+
+.product-card__price-word {
+  font-size: 12px;
+  font-weight: 400;
+  color: var(--secondary-color);
+}
+
+.product-card__price-value {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.product-card__cta-button {
+  margin: 16px 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 166px;
+  height: 40px;
+
+  background-color: var(--button-color);
+  border: 1px transparent;
+  border-radius: 5px;
+
+  text-transform: uppercase;
+  text-decoration: none;
+
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  color: #fff;
+}
+
+.product-card__cta-button:hover {
+  background-color: #fff;
+  border: 1px solid var(--button-color);
+  border-radius: 5px;
```

Comment:
why do we need to change it on hover?

Reply:
![image](https://user-images.githubusercontent.com/121955055/216775275-0f35ebc6-1713-461a-9689-69b4a7cb9da7.png)
It's noted in the task to use :hover pseudo-class